src/include/btmem.h

```diff
@@ -226,33 +222,31 @@ struct __wt_ovfl_reuse {
  * the row-store key is relatively large.
  */
 #ifdef HAVE_BUILTIN_EXTENSION_SNAPPY
-#define	WT_LOOKASIDE_COMPRESSOR	"snappy"
+#define WT_LOOKASIDE_COMPRESSOR "snappy"
 #else
-#define	WT_LOOKASIDE_COMPRESSOR	"none"
+#define WT_LOOKASIDE_COMPRESSOR "none"
 #endif
-#define	WT_LAS_CONFIG							\
-    "key_format=" WT_UNCHECKED_STRING(QIQu)				\
-    ",value_format=" WT_UNCHECKED_STRING(QQQBBu)			\
-    ",block_compressor=" WT_LOOKASIDE_COMPRESSOR			\
-    ",leaf_value_max=64MB"						\
-    ",prefix_compression=true"
+#define WT_LAS_CONFIG                                                                              \
```

Comment:
The formatter doesn't handle macros well if they don't look like real C.
The server had this problem too but WiredTiger gets more mileage out of the preprocessor so it'll be more of a problem.

Reply:
In some places we may be able to fix this by switching to inlined functions. That's better for debugging, anyway.